testdata/p4_16_samples/issue1653-bmv2.p4

```diff
@@ -43,10 +43,9 @@ control ingress(inout parsed_packet_t h, inout local_metadata_t local_metadata,
     bitvec_hdr bh;
 
     apply {
-
         // Copy another header's data to local variable.
         bh.x = h.bvh0.x;
-        clone3(CloneType.I2E, 0, h);
+        clone_preserving_field_list(CloneType.I2E, 0, 0);
```

Comment:
It seems like if we wanted to remain faithful to the original meaning of this test program, there should be `@field_list(0)` annotations on all fields of struct `parsed_packet_t`?

Reply:
The clone only preserves metadata fields,. Adding annotations to parsed_packet_t should have no effect.